provider/datagen/src/transform/cldr/datetime/symbols.rs

```diff
@@ -64,7 +64,7 @@ fn get_month_code_map(calendar: &str) -> &'static [TinyStr4] {
         "gregory" | "buddhist" | "japanese" | "japanext" | "indian" | "persian" | "roc" => {
             &SOLAR_MONTH_CODES[0..12]
         }
-        "coptic" | "ethiopic" => SOLAR_MONTH_CODES,
+        "coptic" | "ethiopic" | "chinese" => SOLAR_MONTH_CODES,
```

Comment:
issue: this is incorrect, we need to be able to handle leap months as well during formatting

Reply:
I think the way CLDR stores it is to have a separate "leap" pattern. We should construct the strings ourselves from the leap map